src/Arcus.Messaging.Health/Tcp/TcpHealthServer.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using GuardNet;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Arcus.Messaging.Health.Tcp
+{
+    /// <summary>
+    /// Representing a TCP server as a background process to expose an endpoint where the <see cref="HealthReport"/> is being broadcasted.
+    /// </summary>
+    public class TcpHealthServer : BackgroundService
+    {
+        private readonly HealthCheckService _healthService;
+        private readonly TcpListener _listener;
+        private readonly ILogger<TcpHealthServer> _logger;
+        private readonly JsonSerializerSettings _serializerSettings;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TcpHealthServer"/> class.
+        /// </summary>
+        /// <param name="configuration">The configuration to control the hosting settings of the TCP server.</param>
+        /// <param name="healthService">The service to retrieve the current health of the application.</param>
+        /// <param name="logger">The logging implementation to write diagnostic messages during the running of the TCP server.</param>
+        public TcpHealthServer(IConfiguration configuration, HealthCheckService healthService, ILogger<TcpHealthServer> logger)
+        {
+            Guard.NotNull(configuration, nameof(configuration), "Requires a configuration implementation to retrieve hosting information for the TCP server");
+            Guard.NotNull(healthService, nameof(healthService), "Requires a health service to retrieve the current health status of the application");
+            Guard.NotNull(logger, nameof(logger), "Requires a logger implementation to write diagnostic messages during the running of the TCP server");
+
+            _healthService = healthService;
+            _logger = logger;
+
+            int tcpPort = GetConfiguredTcpPort(configuration);
+            _listener = new TcpListener(IPAddress.Any, tcpPort);
+            _serializerSettings = CreateDefaultSerializerSettings();
+        }
+
+        private static int GetConfiguredTcpPort(IConfiguration configuration)
+        {
+            const string arcusTcpPort = "ARCUS_TCP_PORT";
```

Comment:
I'd go with `HEALTH` to explicitly show that it will only be used for that
```suggestion
            const string arcusTcpPort = "ARCUS_HEALTH_PORT";
```